### PR TITLE
HTML-CSS-Popup

### PR DESCRIPTION
**Pull request by natayakovleva**
### HTML-CSS-Popup

[Demo](https://natayakovleva.github.io/HTML-CSS-popup/) |
[Code base](https://github.com/natayakovleva/HTML-CSS-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by natayakovleva** [comment]
Hello. Sorry for not answering immediately. I thought the answer would come to the mail. And I was busy with the next task. 
Tell me please, now I need to modify the lists according to the recommendations and re-send the PR again?

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
About what to do next: 
1. Make changes to your repository(not this one) with popup task.
2. Update demo on GitHub pages.
3. Update files in current PR with files from your repository(item 1).
4. Push changes, it automatically updates PR.

**Comment by natayakovleva** [comment]
Hi. Thank you for your comments. I hope I fixed everything)

File: submissions/natayakovleva/HTML-CSS-popup/index.html [hunk 1]
```diff
@@ -11,95 +11,55 @@
...
+                    <li class="popup__item popup__item-chat"><a class="popup__item-link" href="#">Chat</a></li>
+                    <li class="popup__item popup__item-classroom"><a class="popup__item-link" href="#">Classroom</a></li>
+                    <li class="popup__item popup__item-collection"><a class="popup__item-link" href="#">Collection</a></li>
+                    <li class="popup__item popup__item-mars"><a class="popup__item-link" href="#">Mars</a></li>
+                    <li class="popup__item popup__item-blogs"><a class="popup__item-link" href="#">Blogs</a></li>
+                    <li class="popup__item popup__item-promo"><a class="popup__item-link" href="#">Promos</a></li>
+                  </ul>
+                <div class="popup__text">
+                  <p>Even more from Kottans</p>
+                </div>
```
Comment (A-Ostrovnyy) [inline submissions/natayakovleva/HTML-CSS-popup/index.html:55]: Logically it's a link)

File: submissions/natayakovleva/HTML-CSS-popup/index.html [hunk 2]
```diff
@@ -11,95 +11,55 @@
...
-          </div>
-        </div>
-        <a href="#" class="header__link">
-          <img class="header__link-img" src="./img/bell.png" alt="link-bell">
-        </a>  
-        <a href="#" class="header__link">
-          <img class="header__link-img" src="./img/user.png" alt="link-user">
-        </a>
+          </li>
+          <li><a href="#" class="header__link"><img class="header__link-img" src="./img/bell.png" alt="link-bell"></a></li>
```
Comment (A-Ostrovnyy) [inline submissions/natayakovleva/HTML-CSS-popup/index.html:60]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!

**Comment by natayakovleva** [comment]
Hi. I made changes and thanks for the article)

File: submissions/natayakovleva/HTML-CSS-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                    <li class="popup__item popup__item-mars"><a class="popup__item-link" href="#">Mars</a></li>
+                    <li class="popup__item popup__item-blogs"><a class="popup__item-link" href="#">Blogs</a></li>
+                    <li class="popup__item popup__item-promo"><a class="popup__item-link" href="#">Promos</a></li>
+                  </ul>
+                  <a href="#" class="popup__link">Even more from Kottans</a>  
+                </div>
+              </div>
+            </div>
+          </li>
+          <li><a href="#" class="header__link"><img class="header__link-img" src="./img/bell.png" alt="Go to notifications page"></a></li>
```
Comment (A-Ostrovnyy) [inline submissions/natayakovleva/HTML-CSS-popup/index.html:58]: One of the points in the provided article is no need to populate `alt` attribute for decorative images. Icon is a decorative image)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One last issue)

**Comment by natayakovleva** [comment]
Hi and thanks for your help.
Honestly, I was thinking of it as a link, not a button, so I didn’t think to delete it(
Sorry for the delay - family problems

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Comment by natayakovleva** [comment]
Thanks for your help))

**Pull request merged**